.swiftpm/xcode/xcshareddata/xcschemes/SUITCase.xcscheme

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file in the repository?

Reply:
hard to say, if you think we don't need this file I can add it to .gitignore